### DISCUSSION TITLE
Gitlab oauth callback URI

### DISCUSSION THREAD

**Discussion by DocSbrz** [uxbox/uxbox#1002] [Q&A] [answered]
Hi, I'm trying to setup the penpot authentication through the Gitlab oauth service. 
Actually I have set all env about the Gitlab application and penpot is doing it's job, like I have the button and clicking on it the browser is redirecting me to the gitlab authentication page. 
But I can't find the penpot callback uri that Gitlab should invoke when authenticating the user, it is necessary for the gitlab application to go back authenticated to penpot. 
Can someone help me with this please?
Thank you.

**Comment by niwinz** [comment, marked as answer]
Hi @DocSbrz 
The callback is `http://[your-domain]/api/auth/oauth/gitlab/callback`
We have a big TODO on our tasks for document it properly.

**Reply by DocSbrz** [reply]
Thank you!
